frontend/hub/tasks/Task.tsx

```diff
@@ -4,7 +4,7 @@ export interface Task {
   pulp_created: string;
   started_at: string;
   finished_at: string;
-  state: 'completed' | 'failed';
+  state: 'completed' | 'failed' | 'running';
```

Comment:
```suggestion
  state: 'completed' | 'failed' | 'running' | 'waiting';
```

waiting tasks can also be stopped